pool/src/types.rs

```diff
@@ -144,7 +144,7 @@ fn default_max_stempool_size() -> usize {
 	50_000
 }
 fn default_mineable_max_weight() -> usize {
-	global::max_block_weight()
+	consensus::MAX_BLOCK_WEIGHT
```

Comment:
Why?

Reply:
Ah good catch. This was necessary because max_block_weight() relies on `chain_type` internally. So we have a default value that cannot be used until we know (and have configured) the chain_type itself.
The default config needs to be genuine defaults, without any dependencies on other, possibly external, configuration.